docs/filters.rst

```diff
@@ -0,0 +1,87 @@
+.. _filters:
+
+Filtering
+=========
+
+REST resource filtering is provided via method ``_filter_queryset``. If you want implement your own filter technique
+you can add here your own code. Input params of the method are only ``qs`` which represents data that you want to filter
+(for model resources it is django QuerySet object).
+
+Better way is used pre-implemented managers defined in model ``BaseModelResource``::
```

Comment:
A better way is to use pre-implemented managers defined in model ``BaseModelResource``::